charts/minio/values.yaml

```diff
@@ -17,7 +17,11 @@ minio:
     repository: docker.io/minio/minio
     tag: RELEASE.2021-08-20T18-32-01Z
   storeSize: 100Gi
-
+  
+  # The is required to enable the BackupRestore controller so it can backup
+  # and restore from the local minio deployment
+  certificateSecret: minio-certificates # tls secret used by minio.
```

Comment:
This is where we need to comment/document that the user needs to but the CA used for their cert into the global CA bundle. Nobody will ever read the release note ;)